uniprotkb-rest/pom.xml

```diff
@@ -227,35 +227,6 @@
 			<version>${uniprot-store.version}</version>
 			<scope>test</scope>
 		</dependency>
-		<dependency>
```

Comment:
Thank you for cleaning this and deleting  ExampleRedisTestContainerTest. Please make sure that all integ tests are passing locally.